### DISCUSSION TITLE
Pesquisa de campo sobre estudos!

### DISCUSSION THREAD

**Discussion by gabriel-brito** [frontendbr/forum#1249] [Dicas]
Opa, galera, boa tarde!

Da ultima vez, compartilhei com vocês `algumas dicas valiosas para estudantes desesperados` (https://medium.com/trainingcenter/algumas-dicas-valiosas-para-estudantes-desesperados-f644313f4ff4) e mais uma vez desejo compartilhar conteúdo com vocês!

Mas preciso de um auxilio de cada membro da comunidade desta vez, pois necessito que preencham o formulário abaixo, pois assim poderei estruturar um conteúdo mais robusto para disponibilizar.

Poderiam me ajudar preenchendo o formulário abaixo?!

Agradeço a participação de cada um desde já!

 https://docs.google.com/forms/d/1VqH3jeCtA3FQblN7mMa1nbqLgZ4DUFU9F2t02lMVumM/

**Comment by Roger-Melo** [comment]
respondido!

**Comment by evelew** [comment]
respondido :)

**Comment by tunnes** [comment]
Feito.
